tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+import pytest
```

Comment:
We should create a flask app template repo! That way we don't have to rewrite this conftest.py or any other pieces of very general code again 💘